cmd/influxd/run.go

```diff
@@ -231,52 +234,76 @@ func parseConfig(path, hostname string) *Config {
 }
 
 // creates and initializes a broker.
-func openBroker(path string, u *url.URL, initializing bool, joinURLs []*url.URL, w io.Writer) *influxdb.Broker {
+func openBroker(path string, u url.URL, initializing bool, joinURLs []url.URL, w io.Writer, raftTracing bool) (*influxdb.Broker, *raft.Log) {
+	// Create raft log.
+	l := raft.NewLog()
```

Comment:
So it seems like Raft is no longer hidden inside a broker. Was this required by this change, or something that just made the whole thing cleaner?